hack/tests/ci-operator-prepare.sh

```diff
@@ -15,7 +15,8 @@ set_build_images() {
 }
 
 setup_secrets() {
-    ln -s /usr/local/e2e-secrets/azure secrets
+    ln -s $SECRET_DIR secrets
```

Comment:
Would it be useful to say "ln -sf ..." here just in case the setup (especially in dev mode) already has a link created?

Reply:
ok, good idea